msu/config/items.nut

```diff
@@ -37,16 +37,12 @@ foreach (itemType in ::Const.Items.ItemType)
 		return;
 	}
 
-	local max = 0;
-	foreach (w, value in ::Const.Items.ItemType)
-	{
-		if (value > max) max = value;
-	}
+	foreach (itemType, value in ::Const.Items.ItemType) ::Const.Items.ItemType[itemType] = value << 1;
```

Comment:
Won't this cause `None = 0` to become `None = 1`?

Reply:
No, because there are no 1 bits to shift into a 1 (it didn't happen during testing either)